x-pack/elastic-agent/pkg/agent/install/installed.go

```diff
@@ -58,6 +60,15 @@ func RunningInstalled() bool {
 		execDir = filepath.Dir(filepath.Dir(execDir))
 		execPath = filepath.Join(execDir, execName)
 	}
+
+	// Windows filesystem is case insensitive and the following paths are the same.
+	// - C:\TMP
```

Comment:
maybe we can extract this to paths package and compare paths using paths.AreEqual or something similar